eth2util/deposit/deposit.go

```diff
@@ -67,19 +67,24 @@ func getMessageRoot(pubkey eth2p0.BLSPubKey, withdrawalAddr string) (eth2p0.Root
 }
 
 // MarshalDepositData serializes a list of deposit data into a single file.
-func MarshalDepositData(msgSigs map[eth2p0.BLSPubKey]eth2p0.BLSSignature, withdrawalAddr, network string) ([]byte, error) {
-	creds, err := withdrawalCredsFromAddr(withdrawalAddr)
-	if err != nil {
-		return nil, err
-	}
-
+func MarshalDepositData(msgSigs map[eth2p0.BLSPubKey]eth2p0.BLSSignature, withdrawalAddrByPubkey map[eth2p0.BLSPubKey]string, network string) ([]byte, error) {
```

Comment:
should we just pass slices of pubkeys, signatures and withdrawal addresses? rather than passing 2 maps

Reply:
yeah

Reply:
done